src/components/Card/Card.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+interface CardProps {
+    /** ID  */
+    id?: string;
+
+    /** ClassName that appears in addition to "card" */
+    ClassName?: string[];
+
+    /** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    modifiers?: string[];
+
+    /** Optional blockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+
+    /** Heading that appears within the card but above the three columns within it */
+    heading?: React.ReactNode;
+
+    /** Footer that appears within the card but below the three columns within it */
+    footer?: React.ReactNode;
+
+    /** First column within the card */
+    image?: React.ReactNode;
+
+    /** Main content within the card */
+    content?: React.ReactNode;
```

Comment:
I think you need to remove this if you're switching to children : )

Reply:
🤦🏻‍♀️ thank you